dotnet/src/SemanticKernel/Text/TextChunker.cs

```diff
@@ -259,10 +273,8 @@ private static (List<string>, bool) Split(ReadOnlySpan<char> input, string? inpu
         return (result, inputWasSplit);
     }
 
-    private static int TokenCount(int inputLength)
+    private static int FallbackTokenCounter(string input)
```

Comment:
Can you please rename this to "DefaultTokenCounter"?

Reply:
Done! Thanks for the tip, that method name is much better than "Fallback".